config.py

```diff
@@ -64,7 +64,7 @@ class Config:
         "USER_READ_TOKEN_EXPIRATION", timedelta(days=7)
     )
     HMAC_SIGNING_KEY = os.environ.get("HMAC_SIGNING_KEY")
-    USER_CONTROL_HISTORY_DEPTH = timedelta(days=28)
+    USER_CONTROL_HISTORY_DEPTH = timedelta(days=56)
```

Comment:
Est-ce qu'il faudra changer la valeur le 31/12 ? Si oui, peut être utiliser une variable d'environnement qu'on aura juste à changer avant de redémarrer le container ?

```suggestion
        USER_CONTROL_HISTORY_DEPTH = timedelta(
        days=os.environ.get("USER_CONTROL_HISTORY_DEPTH", 28)
    )
```

Reply:
J'ai modifié et rajouté une var d'env et la valeur par défaut à 28, j'ai fait pareil côté front.